eclipse/mars/gcp-eclipse-mars.tpd

```diff
@@ -59,3 +59,7 @@ location "http://download.eclipse.org/tools/orbit/downloads/drops/R2016022119215
     ch.qos.logback.slf4j
     org.slf4j.log4j
 }
+
+location "http://dadacoalition.org/yedit/" {
+  org.dadacoalition.yedit
```

Comment:
Pull down `org.dadacoalition.yedit.feature.feature.group`

Reply:
What do you mean by "pull down"? Change org.dadacoalition.yedit to org.dadacoalition.yedit.feature.feature.group on line 61?